yoti_python_sdk/doc_scan/session/create/notification_config.py

```diff
@@ -16,21 +17,22 @@ class NotificationConfig(YotiSerializable):
     to poll for the state of the Session.
     """
 
-    def __init__(self, auth_token, endpoint, topics=None):
+    def __init__(self, auth_token, endpoint, auth_type=None, topics=None):
```

Comment:
is the default parameter needed for `auth_type`? It looks like `NotificationConfig(...)` is always called with a an `auth_type` param.

Reply:
fixed

Reply:
Usually we would put new parameters on the end of the parameter list: if a caller of this function is passing in the `topics` parameter, after this update they will be passing `topics` as `auth_type` without changing their code. 

Not 100% sure it's necessary here, but would be good if we can do it anyway. Although we shepherd clients toward the builder, I assume they can use this function directly too can't they? :+1: